components/frontend/src/utils.js

```diff
@@ -337,6 +337,13 @@ export function pluralize(word, count) {
     return word + (count === 1 ? "" : "s")
 }
 
+export function limitTextLength(text, maxLength = 250) {
+    if (typeof text === "string" && maxLength > 3 && text.length > maxLength) {
```

Comment:
So `if maxLength < 4`, then the whole content is displayed anyway?

Reply:
Having a max length smaller than "..." doesn't make sense so I was trying to work around that. I guess I should just throw an exception if the max length is smaller than the length of the ellipsis.

Reply:
Fixed by just returning "..." if the maxLength is equal or smaller than the ellipsis length.